vkcs/kubernetes/data_source_cluster.go

```diff
@@ -220,6 +225,7 @@ func dataSourceKubernetesClusterRead(ctx context.Context, d *schema.ResourceData
 	d.Set("loadbalancer_subnet_id", c.LoadbalancerSubnetID)
 	d.Set("insecure_registries", c.InsecureRegistries)
 	d.Set("dns_domain", c.DNSDomain)
+	d.Set("sync_security_policy", c.SecurityPolicySyncEnabled)
```

Comment:
I don't remember, is it OK to set boolean from nil in SDK?

Reply:
Yes, it's ok: https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/resource_data.go#L218